src/ontology/mondo-edit.obo

```diff
@@ -348133,20 +348135,20 @@ intersection_of: has_characteristic MONDO:0024495 ! tumor grade 1 or 2, general
 
 [Term]
 id: MONDO:0015068
-name: neuroendocrine tumor of rectum, well differentiated, low or intermediate grade
+name: rectal neuroendocrine tumor
 def: "A well differentiated, low or intermediate grade tumor with neuroendocrine differentiation that arises from the rectum." [NCIT:C135213]
 subset: gard_rare {source="MONDO:GARD"}
 subset: rare
-synonym: "neuroendocrine tumor of the rectum" RELATED [Orphanet:100081]
-synonym: "neuroendocrine tumour of the rectum" RELATED OMO:0003005 []
-synonym: "rectal neuroendocrine tumor" EXACT [NCIT:C135213, Orphanet:100081]
-synonym: "rectal neuroendocrine tumour" EXACT OMO:0003005 []
+synonym: "neuroendocrine tumor of rectum, well differentiated, low or intermediate grade" EXACT [http://purl.obolibrary.org/obo/MONDO_0015068]
+synonym: "rectal well differentiated neuroendocrine neoplasm" RELATED [https://orcid.org/0000-0002-0587-4693, NCIT:C135213]
+synonym: "rectal well-differentiated neuroendocrine neoplasm" RELATED [https://orcid.org/0000-0002-0587-4693, NCIT:C135213]
```

Comment:
Same as above: this should be exact according to [NCIT:C135213](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes?short_form=NCIT_C135213)

Note: it makes sense because they use the word "tumor" for "well differentiated neoplasm"